src/scss/objects/_form-controls.scss

```diff
@@ -0,0 +1,230 @@
+/**
+ * Custom Checkboxes and radios
+ *
+ * --------------------------------
+ *
+ * Checkbox button example HTML
+ *
+ *    <label class="o-formControl-label" for="checkbox1">
+ *        <input class="o-formControl-input" type="checkbox" id="checkbox1" name="myCheckbox" value="A value"/>
+ *        <span class="o-formControl-indicator o-formControl-indicator--checkbox"></span>
+ *        My Checkbox Label Text
+ *    </label>
+ *
+ * -------------------------------
+ *
+ * Radio button example HTML
+ *
+ *    <label class="o-formControl-label" for="radio1">
+ *        <input class="o-formControl-input" type="radio" id="radio1" name="myRadioBtn" value="A value"/>
+ *        <span class="o-formControl-indicator o-formControl-indicator--radio"></span>
+ *        My Radio Button Label Text
+ *     </label>
+ *
+ * ================================= */
+ $formControl-bgColor           : $white !default;
+ $formControl-bgColor-disabled  : $color-disabled !default;
+ $formControl-borderColor       : $color-border !default;
+ $formControl-color             : $blue !default;
+ $formControl-background-color  : $color-bg !default;
+ $formControl-hover-color       : $blue--dark !default;
+ $formControl-borderColor-hover : $blue !default;
+ $formControl-margin-left       : 0 !default;
+ $formControl-padding-left      : spacing(x4) !default;
+ $formControl-width--wide       : 20px !default;
+ $formControl-width             : 24px !default;
+ $formControl-height--wide      : 20px !default;
+ $formControl-height            : 24px !default;
+ $formControl-border-width      : 1px;
+
+
+ .o-formControl-label {
+     position: relative;
+     display: block;
+     cursor: pointer;
+     user-select: none;
+     padding: spacing();
+     padding-left: $formControl-padding-left;
+
+     &:hover .o-formControl-indicator {
+         border-color: $formControl-borderColor-hover;
+     }
+
+     &:hover .o-formControl-input:disabled ~ .o-formControl-indicator {
+         border-color: $formControl-borderColor;
+         cursor: not-allowed;
+     }
+
+     .o-formControl-input:disabled ~ .o-formControl-indicator {
+        background-color: $formControl-bgColor-disabled;
+     }
+ }
+
+ // Give inline elements some space to the right
+ .o-formControl-label--inline {
+     display: inline-block;
+     padding-right: 2em;
+ }
+
+     // Hide the input element from view
+     .o-formControl-input {
+         position: absolute;
+         opacity: 0;
+         z-index: -1; // Put the input behind the label so it doesn't overlay text
```

Comment:
It would be really useful to have this added to [here](https://github.com/justeat/fozzie/blob/master/src/scss/settings/_variables.scss#L113) maybe as `lowest: -1` I can see it being useful in other areas.